local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -938,6 +940,7 @@ int main(int argc, char** argv) {
   bool landing = false;
   avoidanceOutput planner_output;
   bool startup = true;
+  Node.status_msg_.state = 1; //COMPANION_PROCESS_STATE_STARTING
```

Comment:
As soon as these are updated to the real enums I am happy with this.

Reply:
you are right this is ugly. The enum is already fixed and we know what it looks like. But it is defined in mavlink, therefore I now defined the same enum also in the local_planner such that we can use the same terms